pkg/mount/mount.go

```diff
@@ -57,18 +57,25 @@ func Unmount(target string) error {
 	if mounted, err := Mounted(target); err != nil || !mounted {
 		return err
 	}
-	return ForceUnmount(target)
+	return ensureUnmount(target, 0)
 }
 
-// ForceUnmount will force an unmount of the target filesystem, regardless if
-// it is mounted or not.
-func ForceUnmount(target string) (err error) {
```

Comment:
Note I removed this entirely because it's just weird, and doesn't actually do a force unmount (e.g. `MNT_FORCE`).
Meanwhile changing it to use `MNT_FORCE` seems worse than breaking the package API.